model/Dataset/Vocabulary/DatasetAvailability.md

```diff
@@ -0,0 +1,24 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# DatasetAvailability
```

Comment:
The name of the vocabulary is the same as that of the property, which will likely lead to confusion. It seems to be common to append "Type" at the end of the vocabulary in that case, i.e. `DatasetAvailabilityType`.

Reply:
Also think renaming with append of "Type" will make it clearer.  Good idea

Reply:
New version with Type added.